openstack/testing/endpoint_location_test.go

```diff
@@ -178,6 +178,30 @@ var catalog3 = tokens3.ServiceCatalog{
 				},
 			},
 		},
+		tokens3.CatalogEntry{
+			Type: "someother",
+			Name: "someother",
+			Endpoints: []tokens3.Endpoint{
+				tokens3.Endpoint{
+					ID:        "1",
+					Region:    "someother",
```

Comment:
Note that `Region` here is intentional to test that both `Region` and `RegionID` are working.